### PR TITLE
Add info on how to install with Homebrew

### PR DESCRIPTION
**Pull request by kenden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ govendor sync
 go install
 ```
 
+### Homebrew
+
+On macOS, you can also install kubetail using [Homebrew](https://brew.sh/):
```
Comment (hatchan) [inline README.md]: Replace kubetail with stern

File: README.md [hunk 2]
```diff
@@ -28,6 +28,13 @@ govendor sync
...
+```
+$ brew install stern
```
Comment (hatchan) [inline README.md]: Can you remove the `$` prefix

File: README.md (hunk 1 above)
Comment (kenden) [reply README.md]: :)  Done, thanks for spotting this.

**Comment by kenden** [comment]
@hatchan Fixed and rebased.

**Comment by hatchan** [comment]
Thanks

**Pull request merged**